### ISSUE TITLE
[lightbox-media-viewer - with carousel]: QA: The label "previous page"/"next page" does not seem appropriate. 

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#8915]
### Description

Page Element:
Label for Previous/next buttons

Issue Description:
Since there is no page navigation, The label "previous page"/"next page" does not seem appropriate. 

Expected:
The label can be given as "previous slide"/"next slide" respectively

User Impact:
If the label is incorrect or inadequately descriptive, it affects the screen reader user’s ability to understand the purpose of the control.   

### Component(s) impacted

lightbox-media-viewer - with carousel

### Browser

Safari, Firefox

### Carbon for IBM.com version

canary

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://ibmdotcom-webcomponents.s3.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/8757/iframe.html?id=components-lightbox-media-viewer--with-carousel

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by gopinathan-n** [comment]
Now Screen reader announces the label as "previous"/"next". Issue is fixed
